spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindingUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binding;
+
+import java.util.Map;
+
+import org.springframework.context.ConfigurableApplicationContext;
+
+/**
+ * Utility class for handling the {@link Bindable}s in a given context.
+ *
+ * @author Marius Bogoevici
+ */
+public class BindingUtils {
+
+	public static void bindAll(ConfigurableApplicationContext context) {
```

Comment:
It would be nice to set all these static methods' accessor to `package` after moving the class `UnbindOnCloseApplicationListener` to `binding` package.